OpenSSL/test/util.py

```diff
@@ -105,7 +106,8 @@ def format_leak(p):
             # /home/exarkun/Projects/cpython/2.7/python(PyEval_EvalFrameEx+0x753b) [0x4d0e1e]
             # /home/exarkun/Projects/cpython/2.7/python() [0x4d6419]
             # ...
-            #
+            """  # noqa
```

Comment:
Can we remove the quotes from this section, leave it failing, and come back to it at the end, I want some time to think about the best approach here.